static/css/styles.css

```
@@ -745,6 +745,111 @@ body.fake-mode-active .toast-stack {
+  border-top: 1px solid var(--border-subtle);
+  color: var(--text-muted);
+  font-size: var(--text-sm);
+}
+
+.not-found-path span {
+  flex: 0 0 auto;
+}
+
+.not-found-path code {
+  color: var(--text-secondary);
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)